main.js

```diff
@@ -220,6 +227,28 @@ define(function (require, exports, module) {
         }
     }
     
+    function _applyColors() {
+        // Slight delay so Brackets can apply the dark class
+        setTimeout(function() {
+            var userColors = _preferences.get("colors"),
+                toApply    = $("body").hasClass("dark") ? userColors.dark : userColors.light,
+                css        = ".CodeMirror .cm-dk-whitespace-space:before," +
```

Comment:
Let me look those up. ;)

What I really wish we had was [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings) (good write up [here](http://davidwalsh.name/javascript-template-strings)), but alas those are only implemented in Chrome 41+.